static/app/components/feedback/feedbackItem/feedbackActions.tsx

```diff
@@ -29,73 +25,139 @@ export default function FeedbackActions({
   className,
   eventData,
   feedbackItem,
+  size,
   style,
 }: Props) {
-  const organization = useOrganization();
-  const hasSpamFeature = organization.features.includes('user-feedback-spam-filter-ui');
-
-  const {markAsRead, resolve} = useMutateFeedback({
-    feedbackIds: [feedbackItem.id],
-    organization,
-    projectIds: [feedbackItem.project.id],
-  });
-
-  const mutationOptions = {
-    onError: () => {
-      addErrorMessage(t('An error occurred while updating the feedback.'));
-    },
-    onSuccess: () => {
-      addSuccessMessage(t('Updated feedback'));
-    },
-  };
-
-  // reuse the issues ignored category for spam feedbacks
-  const isResolved = feedbackItem.status === GroupStatus.RESOLVED;
-  const isSpam = feedbackItem.status === GroupStatus.IGNORED;
-
   return (
-    <Flex gap={space(0.5)} align="center" className={className} style={style}>
+    <Flex gap={space(1)} align="center" className={className} style={style}>
       <ErrorBoundary mini>
-        <FeedbackAssignedTo feedbackIssue={feedbackItem} feedbackEvent={eventData} />
+        <FeedbackAssignedTo
+          feedbackIssue={feedbackItem}
+          feedbackEvent={eventData}
+          showActorName={['medium', 'large'].includes(size)}
+        />
       </ErrorBoundary>
 
-      <Button
-        onClick={() => {
-          addLoadingMessage(t('Updating feedback...'));
-          markAsRead(!feedbackItem.hasSeen, mutationOptions);
-        }}
-      >
-        {feedbackItem.hasSeen ? t('Mark Unread') : t('Mark Read')}
+      {size === 'large' ? <LargeWidth feedbackItem={feedbackItem} /> : null}
+      {size === 'medium' ? <MediumWidth feedbackItem={feedbackItem} /> : null}
+      {size === 'small' ? <SmallWidth feedbackItem={feedbackItem} /> : null}
```

Comment:
These are all in the file. each one uses the hook to wire up either buttons or dropdown options, depending on size.